Monika After Story/game/script-topics.rpy

```diff
@@ -16165,3 +16165,38 @@ label monika_giving_criticism:
     m 3dsu "Graciously accept you can't change everyone's mind and stay considerate when assessing someone else's work."
     m 3hub "...That'd be Monika's Critique Tip of the Day, ahaha!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_curse_words",
+            category=["advice", "life"],
+            prompt="Curse words",
+            random=False,
+            pool=False
+        )
+    )
+
+label monika_curse_words:
+    m 3etd "Say, {w=0.2}[player], {w=0.2}do you use curse words sometimes?"
+    m 1hksdlb "Ahaha, {w=0,2}I mean, {w=0,2}it's alright if you swear occasionally, {w=0.2}and I don't have anything against it if you really do."
+    m 5rtd "I even heard of some studies stating that using... {w=0.5}a bit stronger language helps easing the pain if you, say, hurt yourself with something by the accident."
+    m 5eta "Or that it really helps dumping your steam if you feel so angry about something."
```

Comment:
Some expression things to keep in mind.

Pose 5 is generally not used mid-topic often as it's a pretty rough transition. It's normally saved for the more intimate moments as well. Personally for this kinda dialogue (explanatory) I'd use poses 1 and 3 (or 4 if you want to really drive a point home)

Additionally, another thing to keep in mind for pose 5 is that we always `dissolve` into and out of it. As I mentioned before, pose 5 is a rough transition since it's such a big shift in movement. As such, it looks a lot better if before going into a line which uses a 5 pose, we `show monika 5<exp> at t11 zorder MAS_MONIKA_Z with dissolve_monika`

And when coming out of pose 5 (if we're going back into upright, we do the same, except replace `5<exp>` with the upright exp

Reply:
Oh, sorry, I had this animation statement it mind, but I totally forgot about it. I'll add it, thank you.

Reply:
Already added that, so I'm resolving this.